pom.xml

```diff
@@ -41,7 +41,7 @@
     <maven.compiler.target>1.8</maven.compiler.target>
 
     <!-- JBoss Tools Integration Stack version -->
-    <jbtis.version>4.4.1.Final</jbtis.version>
+    <jbtis.version>4.4.2.AM1</jbtis.version>
```

Comment:
we didn't discuss about using the SNAPSHOT yet. I would prefer to do it in a separate PR